### ISSUE TITLE
Decommission #vimeo infrastructure

### ISSUE THREAD

**Issue by cbeams** [bisq-network/dao#22]
We have an existing paid Vimeo account at https://vimeo.com/getbitsquare, but we haven't used it much over time. It doesn't justify its costs, and it's furthermore just non-ideal compared to what we can and will do with a properly-managed YouTube presence (see #21).

 - [x] Copy any videos from Vimeo we want to keep to an appropriate place in our new YouTube channel
 - [x] Update https://bisq.network homepage to embed explainer video from YouTube vs Vimeo
 - [x] Cancel billing for Vimeo / close the 'getbitsquare' account
 - [x] Close associated role issue at bisq-network/roles#36
 - [x] Remove Vimeo entry from bisq-network/roles#readme

**Comment by cbeams** [comment]
@ManfredKarrer, please privately share with me the credentials for the https://vimeo.com/getbitsquare account. I'm finishing up creating the Bisq YouTube channel (bisq-network/youtube#2) and want to download the videos from Vimeo and shut down the account so we stop paying for it etc.

Note that I'll be sure to embed the YouTube version of the explainer video on our website before shutting Vimeo down.

**Comment by cbeams** [comment]
Ping, @ManfredKarrer. Could you send the Bitsquare Vimeo credentials to both me and @ripcurlx? thanks.

**Comment by ManfredKarrer** [comment]
Sent by email.

**Comment by cbeams** [comment]
@ripcurlx, as YouTube Admin (bisq-network/roles#56), would you be up for using Manfred's Vimeo credentials (which I can share with you) to, one way or another, clear out all our videos from Vimeo and categorize them under our YouTube channel?

It may be the case that we want to drop some or even many of these videos, i.e. there may not be any value in bringing them over to YouTube. Another option would be to bring everything over to YouTube and mark some or many of them as unlisted.

In any case, you can see the outline of this work I sketched out in the description of this issue. If you're up for it, I'll assign this issue to you. Thanks.

**Comment by cbeams** [comment]
Closing as complete.

@ManfredKarrer, I've downloaded all the videos from our Vimeo account (I just have them in a folder locally, thought that would be better than fully obliterating them. Maybe I'll upload a zip somewhere, GDrive or something).

I've also deleted our Vimeo account, and first I disabled auto-renewal billing, just to be safe:

![image](https://user-images.githubusercontent.com/301810/39628108-46a8af5c-4fa8-11e8-8346-fb0ba8339b7c.png)

![image](https://user-images.githubusercontent.com/301810/39628112-48ed33d2-4fa8-11e8-9c08-2756cc89ff4d.png)

The explainer video has been transferred to YouTube, and per https://github.com/bisq-network/bisq-website/pull/51, it is now being served from the homepage of https://bisq.network.

So all should be well.
